backend/sample.env

```diff
@@ -88,7 +88,10 @@ PROMPT_PORT=3003
 #Prompt Studio
 PROMPT_STUDIO_FILE_PATH=/app/prompt-studio-data
 
-# Structure Tool
+# Structure Tool (Prompt Studio exported tools)
+# Update to use the desired image for structure tool
+# If the image is not present locally, it will be pulled from Dockerhub
+# https://hub.docker.com/r/unstract/tool-structure
```

Comment:
@chandrasekharan-zipstack We might just need to say this is the structure tool image version, and maybe provide the corresponding Docker hub link too.
Rest of it is obvious.